HACKING.md

```diff
@@ -141,17 +141,82 @@ appropriately.  To install `godep` locally run:
 
 If you are not updating packages you should not need godep installed.
 
-## Updating Godeps from upstream
+## Updating Kubernetes from upstream
+
+There are a few steps involved in rebasing Origin to a new version of Kubernetes. We need to make sure
```

Comment:
Good catch, I did it manually last time so forgot about the script when writing this. I'll add it to the doc.